CHANGELOG.md

```diff
@@ -22,6 +22,34 @@
 - [cronos#550](https://github.com/crypto-org-chain/cronos/pull/550) Support basic json-rpc apis on pruned nodes.
 - [cronos#549](https://github.com/crypto-org-chain/cronos/pull/549) Use custom tx indexer feature of ethermint.
 
+*Aug 5, 2022*
+
+## v0.8.0
+
+### State Machine Breaking
+
+- [#618](https://github.com/crypto-org-chain/cronos/pull/618) selfdestruct don't delete bytecode of smart contract.
+
+*Aug 5, 2022*
+
+## v0.7.1
+
+### Bug Fixes
+
+- [#454](https://github.com/crypto-org-chain/cronos/pull/454) Add back the latest testnet upgrade handler.
```

Comment:
nitpicky but why some of the tag has cronos# prefix and some others dont have?

Reply:
added prefixes